docs/conf.py

```diff
@@ -91,7 +91,7 @@
 rst_epilog += """
 .. |minimum_python_version| replace:: {0.__minimum_python_version__}
 .. |minimum_numpy_version| replace:: {0.__minimum_numpy_version__}
-.. |minimum_scipy_version| replace:: {0.__minimum_scipy_version__}
+.. |minimum_scipy_version| replace:: nonsense
```

Comment:
why is this change?

Reply:
O, darn, that shouldn't have gone in - #10137 - I'll take it out! (and will use `ci skip` since this worked already)